api/models/recipesModel.js

```diff
@@ -0,0 +1,53 @@
+const db = require('../../data/dbConfig.js')
+
+module.exports = {
+    find,
+    findById,
+    findIngredientsById,
+    findDishesById,
+    add,
+    update,
+    remove
+}
+
+function find() {
+    return db('recipes')
+}
+
+function findById(id) {
+    return db('recipes')
+        .where({ id })
+        .first()
+}
+
+function findIngredientsById(recipeId) {
+    return db('recipe_ingredients')
+        .join('ingredients', 'recipe_ingredients.ingredient_id', 'ingredients.id')
+        .select('recipe_ingredients.quantity', 'recipe_ingredients.quantity_unit', 'ingredients.name')
+        .where({ recipe_id: recipeId })
+}
+
+function findDishesById(recipeId) {
+    return db('dishes')
+        .select('dishes.name')
+        .where({ id: recipeId})

```

Comment:
I'm not sure what we're trying to do here. This would get a dish that has the same id of a recipes id